services/app-api/src/emailer/etaTemplates/sendUndoWithdrawnRateStateEmail.eta

```diff
@@ -0,0 +1,17 @@
+<h1 style=<%~'"font-weight: normal;"'%>>Rate <%= it.rateInfo.rateName %> status was updated to 'Submitted' by CMS</h1>
+<b>Updated by:</b> <%= it.updatedBy %><br />
+<b>Updated on:</b> <%= it.updatedOn %><br />
+<b>Reason:</b> <%= it.reason %><br />
+<b>Related submission<%= it.associatedContracts.length > 1 ? 's' : '' %>:</b>
```

Comment:
This code does not add the `s` to submission when there are more than 1 contract. See my screen shot at red box 3